Networking/Extensions/DecodedExtension.swift

```diff
@@ -29,6 +29,17 @@ public extension Decoded {
     
 }
 
+public extension Decodable where Self: RawRepresentable, Self.RawValue == String {
```

Comment:
What other types should we support? the `JSON` type that comes from Argo can either be a `String`, a `NSNumber`, a `Bool`, an array or another json. I think we can support the first 3 automatically and if the user has a custom `RawRepresentable` that needs a more complex decoding then he will have to make a custom decoder.